packages/manager/modules/vrack/src/add/vrack-add.html

```diff
@@ -28,9 +28,9 @@ <h2 class="text-center" data-translate="vrack_validationPending" data-translate-
                 <div data-ng-if="!VrackAddCtrl.loaders.validationPending">
                     <!-- contracts -->
                     <div class="row" data-ng-if="!VrackAddCtrl.model.contractsAccepted && VrackAddCtrl.model.agreements.length"
-                         data-contracts="VrackAddCtrl.model.agreements"
                          data-full-text="true"
-                         data-contracts-validated="VrackAddCtrl.model.contractsAccepted">
+                         data-ovh-contracts="VrackAddCtrl.model.agreements"
```

Comment:
You will have conflict with #1666.

@cbourgois Are there any requirements to prefer using the directive from `cuc` instead of this one?

Reply:
The conflict is fixed @antleblanc .Thanks!